### DISCUSSION TITLE
Denied access to admin only route: /api/job

### DISCUSSION THREAD

**Discussion by dodg3r** [simulot/immich-go#346] [General]
Hi.

I see a lot of these error messages in the logs when uploading photos. 
Did I do something wrong in the configuration or does this happen when the user is not admin?

I´m using 0.18.1 and immich version 1.107.0

Thanks.

**Comment by simulot** [comment]
You're may be right... I haven't checked with regular users

**Reply by dodg3r** [reply]
Ok.
But that access is only for information in the ui?
Or does it affect the upload function?

**Reply by simulot** [reply]
I'll release a fix tonight

**Reply by simulot** [reply]
Only with UI
